tool-plugins/intellij/build.gradle

```diff
@@ -59,30 +59,6 @@ intellij {
     // alternativeIdePath localIdePath
 }
 
-apply plugin: 'de.undercouch.download'
-task downloadPsiViewerPlugin() {
```

Comment:
Why are we removing these tasks? `runIde` task should not have any impact on building the plugin IIRC. Unless we specifically invoke that task (to test), it will not have any impact on building the plugin offline.

Reply:
This downloads beginning at every build even if we compile a single jar. This is because it's written as a generic task not a download task. So you should be able to fix it by 
```groove
task downloadPsiViewerPlugin(type: Download) {
    src 'https://plugins.jetbrains.com/plugin/download?updateId=31087'
    dest new File("${buildDir}/tmp/plugins/", 'PsiViewer.jar')
}

```

Reply:
When trying to build in offline mode, build fails with the following exception.

```gradle
FAILURE: Build failed with an exception.

* Where:
Build file '/home/nino/Desktop/ballerina/tool-plugins/intellij/build.gradle' line: 100

* What went wrong:
A problem occurred evaluating project ':ballerina-intellij-idea-plugin'.
> Unable to download file 'https://plugins.jetbrains.com/plugin/download?updateId=31087' in offline mode.
```

Reply:
@manuranga Your fix works. :) Thanks for the suggestion!